src/app/leaves/add/components/LeaveRequestFormBuilder.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { MembersProps } from "@/types";
+import { LeaveTypes, UserRole, UserStatus } from "@/constant/constant";
+import { LeaveRequestFormBuilder } from "./LeaveRequestFormBuilder";
+
+describe("Leave request Form Testing", () => {
+  const mockMembers: MembersProps = {
```

Comment:
can we use all constant data outside of the describe function

Reply:
Yes, we can use it, but if it's used in the describe function, it won't have any effect.